cmd/loom/chainconfig/cmd.go

```diff
@@ -214,11 +216,42 @@ func ListFeaturesCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			out, err := formatJSON(&resp)
-			if err != nil {
-				return err
+
+			type MaxLength struct {
+				Name        int
+				Status      int
+				Validators  int
+				Height      int
+				Percentage  int
+				BuildNumber int
+			}
+
+			ml := MaxLength{Name: 4, Status: 6, Validators: 10, Height: 6, Percentage: 6, BuildNumber: 5}
+			for _, value := range resp.Features {
+				if len(value.Name) >= ml.Name {
+					ml.Name = len(value.Name)
+				}
+				if len(value.Status.String()) >= ml.Status {
```

Comment:
the condition should be just `>` not `>=`, so that we can save one operation

Reply:
we don't need to check `Status` as it is alway `PENDING` or`WAITING` or `ENABLED`